brawlcord/utils/brawlers.py

```diff
@@ -575,6 +575,17 @@ def super_info(self, stats):
         info = f"{super_desc}\n{emojis['super']} {super_str}: {self.ult['duration']} seconds"
 
         return info
+                        
+class Rosa(Brawler):
+    """Class to represent Rosa."""
+
+    def super_info(self, stats):
+        super_str = self.ult['extra']
+        super_desc = f"```{self.ult['desc']}```"
+
+        info = f"{super_desc}\n{emojis['super']} {super_str}: {self.ult['duration']} seconds"
+
+        return info
 
```

Comment:
Rosa's Super mechanics are identical to Leon's. This is purely a placeholder, I don't know how to impliment a damage reduction mechanic

Reply:
Oops, accidentally made a review. Just pretend it's a comment ;)